src/engine-actions/get-total-commit-fees.js

```diff
@@ -0,0 +1,26 @@
+const { Big } = require('../utils')
+const { listChannels } = require('../lnd-actions')
+
+/**
+ * Get total commit fees for all channels for a specific daemon
+ *
+ * @returns {string} totalCommitFees (int64)
+ */
+async function getTotalCommitFees () {
+  const { channels = [] } = await listChannels({ client: this.client })
+
+  if (channels.length === 0) {
+    this.logger.debug('getCommitFees: No channels exist')
+    return Big(0).toString()
+  }
+
+  // We include commit fees only if the channel was the initiator
```

Comment:
Would be better to include an explanation of why that is rather than just what we're doing.